components/nds-tabs/src/Tabs.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { slugify } from "@nice-digital/nds-core/es/utils";
+import { slugify } from "@nice-digital/nds-core/src/utils";
```

Comment:
This should be es/lib not src shouldn't it as we don't include the src when we package and even if we did, webpack/babel wouldn't transiple a file from within node_modules by default?

Reply:
absolutely, i think webstorm dynamically changed this at some point, i hadn't manually edited it!

Reply:
This is still src not es!